web-api/src/persistence/elasticsearch/docketEntry/getAllPendingMotionDocketEntriesForJudge.test.ts

```diff
@@ -52,7 +52,8 @@ describe('getAllPendingMotionDocketEntriesForJudge', () => {
         parent_type: 'case',
         query: {
           bool: {
-            must: [
+            minimum_should_match: 1,
+            should: [
```

Comment:
i think you might be wanting to use the filter expression on bool. It's like must, but it doesn't score the items.